.jpb/persistence-units.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PersistenceUnitSettings">
+    <persistence-units>
+      <persistence-unit name="Default">
+        <packages>
+          <package value="com.skalashynski.spring.springboot" />
+        </packages>
+      </persistence-unit>
+    </persistence-units>
+  </component>
```

Comment:
Question: let me know, what's the purpose of using persistence-units.xml

Reply:
We don't have any purpose to use this file. It was created automatically when the idea asked to configure the JPA plugin. persistence-units.xml was deleted